### DISCUSSION TITLE
Unable to see Task logs in Airflow Deployed via Kubernetes

### DISCUSSION THREAD

**Discussion by vkandiva** [apache/airflow#16742] [Q&A]
I am unable to view any of the logs I am logging in an instance of Airflow deployed on Kubernetes. For e.g. I was using logging.info('Log Started') to log something and it never shows up in the UI. Even in a test Dag with just logs in it I couldnt see any logs.

Was currently using Airflow 2.0.2 release and installing it via its latest Helm chart using the Kubernetes executor.

I also had to set "AIRFLOW__KUBERNETES__DELETE_WORKER_PODS" to false in the helm file to see some of these logs since my workers were completing but I cant see anything I try to log in the Airflow UI.

What else do I need to set to see the logging in my tasks properly.

**Comment by nendrasai1** [comment]
Hi, how you have fixed this. I am facing the similar issue can you please suggest. Thanks in advance!!
